gnosis/cowswap
# [2506] Sync BE errors with FE

# Summary

Closes #2506 - syncs latest errors and descriptions with FE for appropriate handling in the UI

Screenshots
<img width="674" alt="image" src="https://user-images.githubusercontent.com/21335563/156017242-4e285d9e-f480-4070-9762-bc14e3bf5aaf.png">

# To Test
Connect SC wallet
Set native as buy token
Input an amt as sell
Error should appear
